### PR TITLE
Add COPYRIGHT header information to README

### LINKED ISSUE #1: Add spdx string
SPDX are short-form identifiers to communicate FOSS license information
in a simple, efficient, portable and machine-readable manner

https://spdx.dev/ids/

Seems spack for example uses these in headers, with a short copyright referring to main copyright statement
```python
# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
# Spack Project Developers. See the top-level COPYRIGHT file for details.
#
# SPDX-License-Identifier: (Apache-2.0 OR MIT)
```
Speaking of spack, their [COPYRIGHT](https://github.com/spack/spack/blob/develop/COPYRIGHT) has a bit more information, specifically about not having to sign over copyright for contributors. Licensing hurts my head.

Should update the README with this information and include an example copyright statement.

### PR DESCRIPTION
**Pull request by aidanheerdegen**
Add COPYRIGHT header examples to README with SPDX identifier

Fixes #1

### CODE REVIEW

**Review by harshula — COMMENTED** [review]
The document, https://opensource.com/article/20/10/copyright-notices-open-source-software , written by the Red Hat legal team, explains your final point better than the Wikipedia document.
Personally, I would not have put multiple examples of how to write the copyright notice in different programming languages.

File: README.md [hunk 1]
```diff
@@ -5,3 +5,53 @@ A general template repository for default settings when creating new repositorie
...
+
+An example, short, copyright statement is reproduced below, as it might appear in different coding languages. Copy and add to files as appropriate: 
+
+#### plaintext
+```text
+*********************************************************************************************
+* Copyright 2022 ACCESS-NRI and contributors. See the top-level COPYRIGHT file for details. *
+* SPDX-License-Identifier: Apache-2.0                                                       *
+*********************************************************************************************
+```
```
Comment (ccarouge) [inline README.md]: Why is it the only one enclosed between 2 lines of symbols? I'm no big fan of the lines of symbols...

**Review by ccarouge — APPROVED** [review]
I read Harshula's comment and I think the Wikipedia article explain the 50 years better.

I would keep the various example for a quick copy-paste.

File: README.md (hunk 1 above)
Comment (aidanheerdegen) [reply README.md]: No big reason, but all the others are in code, so it is clearly "comment", there is no such thing in a plaintext file, so it is one of the ways of showing that this is not part of the body of the file. Happy to remove it, or replace that with something else that does that job.

Examples here suggest that perhaps a line at the bottom would be better

https://www.termsfeed.com/blog/sample-copyright-notices/

e.g.

© 2022 ACCESS-NRI and contributors. See the top-level COPYRIGHT file for details. SPDX-License-Identifier: Apache-2.0

Comment (aidanheerdegen) [reply README.md]: I've changed to the example above (sort of).

**Pull request merged**